ts/store/reducers/persistedPreferences.ts

```diff
@@ -52,7 +54,8 @@ export const initialPreferencesState: PersistedPreferencesState = {
   isMixpanelEnabled: null,
   isPnTestEnabled: false,
   isIdPayTestEnabled: false,
-  isDesignSystemEnabled: false
+  isDesignSystemEnabled: false,
+  isIdPayInitiativeBannerClosed: {}
```

Comment:
We cannot add this inside the idpay code store since it is a preference that must be persist on the device.

Inside the `ts/boot/configureStoreAndPersistor.ts` as you can see there is the store `features` inside the blacklist array (row `371`), this means that everything is under `features` store (as idpay) cannot be persisted, so I think it's better to add it into `persistedPreferences` store where inside of it there are all settings that are persisted on the device.